### PR TITLE
Nanodrop concentration fix

### PR DESCRIPTION
**Pull request by bwolfe-benchling**
Fix issue where concentration was not properly captured when NA Type was not present as a column in the raw file.  Adjust case sensitivity of column recognition.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Total cells column no longer required for vi-cell XR
 - Ignore invalid first row when present for vi-cell XR files
+- Capture concentration on Nanodrop Eight files that do not have NA Type column
 - Removed hardcoding of date parsing around Gen5 plate numbers
+
```
Comment (bdworth) [inline CHANGELOG.md:15]: Could you please remove this newline? Although I agree this makes things easier to read in plaintext, when viewed in markdown the newline isn't necessary. Not to mention internal style consistency is important.

File: src/allotropy/parsers/thermo_fisher_nanodrop_eight/nanodrop_eight_reader.py [hunk 2]
```diff
@@ -21,6 +21,6 @@ def read(cls, contents: IOType) -> pd.DataFrame:
             # Prevent pandas from rounding decimal values, at the cost of some speed.
             float_precision="round_trip",
         )
-        raw_data = raw_data.rename(columns=lambda x: x.strip())
+        raw_data = raw_data.rename(columns=lambda x: x.strip().lower())
```
Comment (bdworth) [inline src/allotropy/parsers/thermo_fisher_nanodrop_eight/nanodrop_eight_reader.py:24]: This refactor and all the changes associated with it are making it harder to see the real fixes. Could you please either undo this change or split it into a separate PR with this depending on it?

**Review by bdworth — CHANGES_REQUESTED** [review]

File: src/allotropy/parsers/thermo_fisher_nanodrop_eight/nanodrop_eight_reader.py (hunk 2 above)
Comment (bwolfe-benchling) [reply src/allotropy/parsers/thermo_fisher_nanodrop_eight/nanodrop_eight_reader.py:24]: This is necessary for the fix.  Part of the reason concentration was not being captured in test files 1 and 2  was due to inconsistency in header capitalization.  Does it help if the column header capitalization changes and the additional logic for handling the concentration are separate commits?

File: src/allotropy/parsers/thermo_fisher_nanodrop_eight/nanodrop_eight_parser.py [hunk 3]
```diff
@@ -262,44 +270,42 @@ def _get_measurement_document(
...
-
         if _get_str_or_none(data, row, a280_col):
             # capture concentration on the A280 measurement document if the experiment type is
-            # something other than DNA or RNA
+            # something other than DNA or RNA or if the experiment type is not specified
             mass_concentration = None
-            if na_type is not None and "NA" not in na_type and concentration_col:
+            if (na_type is not None and "NA" not in na_type and concentration_col) or (
+                na_type is None and concentration_col
+            ):
```
Comment (slopez-b) [inline src/allotropy/parsers/thermo_fisher_nanodrop_eight/nanodrop_eight_parser.py:284]: Shorter, easier to read statement:
```suggestion
            if concentration_col and ("NA" not in (na_type or "")):
```

File: src/allotropy/parsers/thermo_fisher_nanodrop_eight/nanodrop_eight_reader.py (hunk 2 above)
Comment (bdworth) [reply src/allotropy/parsers/thermo_fisher_nanodrop_eight/nanodrop_eight_reader.py:24]: Yep, totally fine if the column header capitalization changes are a separate PR!

**Review by bdworth — APPROVED** [review]
After trying to split this up in person, fine keeping as is in one PR.

**Review by bdworth — APPROVED** [review]
